tscreen_solaris.go

```diff
@@ -1,6 +1,6 @@
-// +build solaris
+// +build solaris illumos
```

Comment:
Technically listing illumos here isn't necessary, since all illumos systems are also solaris systems (as far as GO is concerned).  I'm going to over look this though.

Reply:
I'm aware, as I did the work to get the build tag into Go itself.  That said, I wouldn't expect the `solaris` tag to hang around forever.  The Go Porting Policy page has a section on what would lead them to [remove a port](https://github.com/golang/go/wiki/PortingPolicy#removing-a-port).  The Oracle folks have given up maintenance as far as I can tell, so it seems like a matter of time before they'll consider removing the `solaris` bits altogether.

As such, I try to put the `illumos` tag next to the `solaris` tag, even though it's technically redundant today.  Hopefully we don't then have to go through a second round of what amounts to `ifdef` fixup later on.

Thanks for taking a look!